transformer_engine/jax/fp8.py

```diff
@@ -159,12 +159,14 @@ def initialize(margin: float = 0.0,
             _format2dtypes(FP8Helper.FP8_FORMAT)
         FP8Helper.UPDATE_FP8META_INTERVAL = update_fp8meta_interval
         FP8Helper.AMAX_HISTORY_SIZE = amax_history_size
+        assert FP8Helper.AMAX_HISTORY_SIZE == 1, \
+            "It only support amax_history_len == 1 for now."
```

Comment:
Could we replace `AMAX_HISTORY_SIZE` -> `AMAX_HISTORY_LEN` and `amax_history_size` -> `amax_history_len` for consistency with the PyTorch API. On the JAX side I see that these ids are being used interchangeably.